tests/unit/libbpf_test.cpp

```diff
@@ -2161,15 +2161,15 @@ TEST_CASE("enumerate link IDs with bpf", "[libbpf]")
     // Now enumerate the IDs.
     memset(&attr, 0, sizeof(attr));
     REQUIRE(bpf(BPF_LINK_GET_NEXT_ID, &attr, sizeof(attr)) == 0);
-    uint32_t id1 = attr.next_id;
+    uint32_t id1 = attr.link_get_next_id.next_id;
```

Comment:
The changes in this file test that this PR does not break functionality, but is there any way to test that the result is Linux ABI compatible?

Reply:
Not right now. My plan is to add a CI step to this repo which fetches the ebpf-go unit tests of the last release and runs those against the code in the PR. For that to happen I need to merge support upstream, which is blocked on getting things merged here. We discussed this in one of the first office hours: https://github.com/microsoft/ebpf-for-windows/issues/3729#issuecomment-2258195646

Reply:
Ok